ide/eclipse/bundles/org.apache.syncope.ide.eclipse.plugin/pom.xml

```diff
@@ -40,6 +40,11 @@ under the License.
       <artifactId>syncope-client-lib</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
jsoup has MIT license, according to its [website](https://jsoup.org/license), which is fine since such license is [compatible with AL 2.0](http://www.apache.org/legal/resolved.html#category-a).

You need, however, to add the appropriate content into `ide/eclipse/LICENSE` and `ide/eclipse/NOTICE`